### ISSUE TITLE
[M1 Chip, macos] It seems like your platform or Python version are not supported yet.

### ISSUE THREAD

**Issue by siilats** [reloadware/reloadium#3]
I Get this:
(venv) xxx/venv/bin/python -m reloadium pydev_proxy /Applications/PyCharm.app/Contents/plugins/python/helpers/pydev/pydevd.py --multiprocess --save-signatures --qt-support=auto --client 127.0.0.1 --port 55863 --file xx/__init__.py --infer_schema

It seems like your platform or Python version are not supported yet.
Please submit a github issue to let us know at https://github.com/reloadware/reloadium

im trying to debug a module

**Comment by dkrystki** [comment]
What OS are you running it on?

**Comment by cakarlen** [comment]
Having the same issue.

I'm tried using Python 3.8/3.10 on macOS 12.3 (Monterey) with M1 chip

**Comment by dkrystki** [comment]
Missing wheels for ARM64 are causing this and it seems like Github actions does not provide environment to build wheels for this platform yet. I'll see if I have any luck with cross compilation.

**Comment by cakarlen** [comment]
Any luck?

**Comment by dkrystki** [comment]
Not much luck with cross compiling and I haven't found ETA for M1 github virtual environments either but it seems like support for self hosted M1 runners is coming soon according to this issue https://github.com/actions/runner/issues/805 
When it's supported I'll get an M1 machine and set up a runner.

**Comment by cakarlen** [comment]
Is there anyway I can compile a wheel for this project? I don't see a `setup.py` in the project structure.

**Comment by loveJesus** [comment]
God is good, mac m1 problem here too, thanks for working on this and blessings in Jesus name

**Comment by siilats** [comment]
I have m1 and I can compile stuff if you have an idea whats missing. Is there a way to turn on logging or test on command line to see which wheel is missing?

**Comment by thommor** [comment]
Im on an M1 chip as well :(

**Comment by anicolaides** [comment]
Same here 👍

**Comment by cakarlen** [comment]
Any updates here?

**Comment by dkrystki** [comment]
Unfortunately still blocked by https://github.com/actions/runner/issues/805 and https://github.com/actions/virtual-environments/issues/2187 
It seems like a lot of packages suffer from this.
I've tried setting up VM locally to at least build it and add some compatibility but currently there is no way to do that as well.

**Comment by daghoidahl** [comment]
I'm trying to build the plugin myself from the `main` branch on a MacBook Pro M1, but it fails.

First I need to remove some misplaced changelog text from `pycharm/src/main/java/rw/profile/ProfileResultToggle.java`. After that, I get this message:

```
❯ gradle buildPlugin                                                                                  ─╯

> Task :compileJava FAILED
/Users/dagh/programs/reloadium/pycharm/src/main/java/rw/handler/sdk/BaseSdkHandler.java:6: error: cannot find symbol
import rw.consts.Const;
                ^
  symbol:   class Const
  location: package rw.consts

  [same error is repeated for a bunch of locations]
```

**Comment by siilats** [comment]
> Unfortunately still blocked by [actions/runner#805](https://github.com/actions/runner/issues/805) and [actions/virtual-environments#2187](https://github.com/actions/virtual-environments/issues/2187)
> It seems like a lot of packages suffer from this.
> I've tried setting up VM locally to at least build it and add some compatibility but currently there is no way to do that as well.

M1 runners are available now in preview!

Hey folks, thank you for your patience while we got this release ready for you!

A pre-release version of the macOS-ARM64 runner is available now in runner version [2.292.0](https://github.com/actions/runner/releases/tag/v2.292.0) or later 🎉 🎉 🎉 .

Please try it out and as always, if you see any problems please open an issue!

**Comment by codefather-labs** [comment]
please add m1 support to the plugin which is in the marketplace. I don't understand what needs to be done to build it myself, I don't understand Java

**Comment by codefather-labs** [comment]
> > Unfortunately still blocked by [actions/runner#805](https://github.com/actions/runner/issues/805) and [actions/virtual-environments#2187](https://github.com/actions/virtual-environments/issues/2187)
> > It seems like a lot of packages suffer from this.
> > I've tried setting up VM locally to at least build it and add some compatibility but currently there is no way to do that as well.
> 
> M1 runners are available now in preview!
> 
> Hey folks, thank you for your patience while we got this release ready for you!
> 
> A pre-release version of the macOS-ARM64 runner is available now in runner version [2.292.0](https://github.com/actions/runner/releases/tag/v2.292.0) or later 🎉 🎉 🎉 .
> 
> Please try it out and as always, if you see any problems please open an issue!

help me please with m1 chip

**Comment by dkrystki** [comment]
Added in reloadium 0.9.2 (PyCharm plugin 0.8.5)
